shub/deploy.py

```diff
@@ -140,17 +141,17 @@ def _upload_egg(target, eggpath, project, version, auth):
     return make_deploy_request(url, data, files, auth)
 
 
-def _build_egg():
+def _build_egg(setup_file):
     closest = closest_scrapy_cfg()
     os.chdir(os.path.dirname(closest))
-    if not os.path.exists('setup.py'):
+    if not os.path.exists(setup_file):
```

Comment:
I'm not sure this is OK for custom setup files. For example, if the user mistypes the setup file name it will silently generate a new setup file based on the template and (may) deploy without errors. 

I think it's a good idea to restrict this feature only when setup_file == 'setup.py'?

Thoughts? @bbotella @eliasdorneles ?